src/vonage/_internal.py

```diff
@@ -13,10 +14,13 @@
 
 
 class BasicAuthenticatedServer(object):
-    def __init__(self, host, user_agent, api_key, api_secret, timeout=None):
+    def __init__(self, host, user_agent, api_key, api_secret, timeout=None, pool_connections=10, pool_maxsize=10, max_retries=3):
         self._host = host
         self._session = session = Session()
         self.timeout = None
+        adapter = HTTPAdapter(pool_connections=pool_connections, pool_maxsize=pool_maxsize, max_retries=max_retries)
```

Comment:
Then here I'd add just the `max_retries=3`. It does seem a bit arbitrary to define it here, but until I have time to refactor the whole module, I think this is best.

Reply:
```suggestion
        self.max_retries = 3
        adapter = HTTPAdapter(max_retries=self.max_retries)
```

Reply:
ok for me here